server/src/main/java/com/server/loan/calculator/validator/AdminDataValidator.java

```diff
@@ -0,0 +1,33 @@
+package com.server.loan.calculator.validator;
+
+import com.server.loan.calculator.exception.ValidationException;
+import com.server.loan.calculator.model.AdminData;
+import io.micrometer.common.util.StringUtils;
+import lombok.RequiredArgsConstructor;
+
+import java.math.BigDecimal;
+
+@RequiredArgsConstructor
+public class AdminDataValidator {
+    private final AdminData adminData;
```

Comment:
should not be here as a field. methods that use/validate data should take it as a parameter